README.md

```diff
@@ -60,11 +47,26 @@ SSB-JS organization require discussion and consent.
 - Projects SHOULD have automated tests for all target platforms.
 - Projects SHOULD have internally consistent contribution guidelines.
 
-### Maintenance
+### Contributing
 
+- Anyone MUST be able to propose changes to repositories via pull request.
+- Anyone who is not a maintainer MUST NOT have the ability to push commits to repositories.
+- Maintainers MUST have the ability to push commits to every repository.
 - Maintainers SHOULD NOT merge their own pull requests.
 - Maintainers SHOULD aim to make contributing a fun and simple experience.
 - Maintainers SHOULD merge project pull requests in a fast and friendly manner.
 - Maintainers SHOULD merge project pull requests that are net positive and pass tests.
 - Maintainers SHOULD NOT suggest small improvements on project pull requests.
 - Maintainers SHOULD suggest their improvements by opening their own pull request.
+- Maintainers SHOULD close pull requests that are inactive for more than 30 days.
+
+### Governance
+
+- The governance repository MUST be hosted at <https://github.com/ssb-js/ssb-js.git>.
```

Comment:
I would say:

```
This governance repository MUST ... 
```

Reply:
Good idea! I'd approve a PR to make that change.